sound/soc/sof/loader.c

```diff
@@ -192,6 +192,28 @@ static int ext_man_get_dbg_abi_info(struct snd_sof_dev *sdev,
 	return 0;
 }
 
+static int ext_man_get_cavs_config_data(struct snd_sof_dev *sdev,
+				   const struct sof_ext_man_elem_header *hdr)
+{
+	const struct sof_ext_man_cavs_config_data *config_data =
+		container_of(hdr, struct sof_ext_man_cavs_config_data, hdr);
+	int i;
+
+	for (i = 0; i < SOF_EXT_MAN_CAVS_CONFIG_LAST_ELEM; i++)
+		switch (config_data->elems[i].token) {
+		case SOF_EXT_MAN_CAVS_CONFIG_CAVS_LPRO:
```

Comment:
@fredoh9 this should also be in the platform-specific code as well. Maybe we need a new op to parse_ext_manifest_config_data in snd_sof_dsp_ops() and add the parsing for the platform data there.